app/controllers/api/requests_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Api::RequestsController < ApplicationController
-  before_action :authenticate_user!, only: [:create]
+  before_action :authenticate_user!, only: %i[create update]
```

Comment:
Did you forget to remove this when you realized you were doing it in another file?

Reply:
done